bindgen/codegen/mod.rs

```diff
@@ -274,6 +274,8 @@ struct CodegenResult<'a> {
     /// that name. This lets us give each overload a unique suffix.
     overload_counters: HashMap<String, u32>,
 
+    overload_counts: HashMap<ItemId, u32>,
```

Comment:
Could you document this field? Specially on why is this needed if `overload_counters` already exists.

Reply:
I think this can be removed again now. It was needed for sorting duringcode generation. Now, sorting is done while parsing, and it seems to work without this change.